public/coffee/ide.coffee

```diff
@@ -127,6 +127,13 @@ define [
 		ide.permissionsManager = new PermissionsManager(ide, $scope)
 		ide.binaryFilesManager = new BinaryFilesManager(ide, $scope)
 
+		# Set up labels
+		$scope.$on 'doc:labels:updated', labels.onDocLabelsUpdated
+		$scope.$on 'entity:deleted', labels.onEntityDeleted
+		$scope.$on 'file:upload:complete', labels.fileUploadComplete
+		$timeout () ->
+			labels.loadProjectLabelsFromServer()
+
```

Comment:
Could these be handled in `LabelsManager` to keep all label related code in one place? Do we need to `rootScope`? That should still be possible in `LabelsManager`